sematic/db/migrations/20230217013045_add_user_id_to_runs.sql

```diff
@@ -0,0 +1,7 @@
+-- migrate:up
+
+ALTER TABLE runs ADD COLUMN user_id character(32) REFERENCES users(email);
```

Comment:
This limits user emails to `32` characters, right? There are probably some users out there whose emails will break that limit.

Reply:
Also, shouldn't this reference the `id` field instead of `email`?